src/sonarwhal-theme/helper/index.js

```diff
@@ -274,6 +275,25 @@ module.exports = function () {
         isPending: (status) => {
             return status === jobStatus.pending;
         },
+        linkify: (msg) => {
+            const regex = /(https?:\/\/[a-zA-Z0-9.\\/?:@\-_=#]+\.[a-zA-Z0-9&.\\/?:@-_=#]*)\s[a-zA-Z]/g;
+            // Modified use of regular expression in https://stackoverflow.com/a/39220764
```

Comment:
Why not use [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) as the [answer](https://stackoverflow.com/a/39220277) even suggests?

Reply:
@alrra We need to extract the url from a message that contains url, not a url string, such as `jQuery@2.1.4 has 2 known vulnerabilities (1 medium, 1 low). See https://snyk.io/vuln/npm:jquery for more information..` So the URL constructor doesn't work.

Reply:
>  So the URL constructor doesn't work.

We could `split` the string using a space and iterate. Problem is if there's  a `,` or similar at the end of the URL, we won't be linking to the right one.

Reply:
Also the case like this: ` File https://www.odysys.com/ … hedule-Your-Demo-Now.png could be around 37.73kB (78%) smaller.`
> Problem is if there's a , or similar at the end of the URL, we won't be linking to the right one.